views/1add-new-job-role.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
[Possible Future Improvement] - As we are using very similar HTML. It would be best to make use of templates. These allow us to reuse code. https://mozilla.github.io/nunjucks/templating.html#template-inheritance:~:text=own%20as%20well.-,Template%20Inheritance,-Template%20inheritance%20is